### ISSUE TITLE
Missing donations box

### ISSUE THREAD

**Issue by kernc** [biolab/orange-web#79]
Besides PayPal, consider other customary means of online support, like:
* PayPal recurring donation,
* [Patreon](https://www.patreon.com/) - has a traction within the online media community,
* [BountySource](https://www.bountysource.com/) - funding specific bugs / features.

In fact, a dedicated "How can you help" page would be great.

**Comment by ajdapretnar** [comment]
@jasminka What do you think?

**Comment by lanzagar** [comment]
Someone mentioned we can accept donations, but not sponsorships (tax reasons).
Maybe the third option falls into the latter category? But the first two options still sound good.

**Comment by ajdapretnar** [comment]
> Someone mentioned we can accept donations, but not sponsorships (tax reasons).
> Maybe the third option falls into the latter category?

You're right. BountySource, while sounding perfect, would cause a lot of issues tax-wise.
We should look into other two though.

**Comment by kernc** [comment]
BountySource also has [bug-independent salary program](https://salt.bountysource.com/) (like Flattr, Gratipay). But more importantly, when the reward is per bug, it's not necessary your incapacitated institution bills it. It can be my natural person in their spare time that fixes it, or any member of the wider community. It's about stewarding the _project_, not necessarily getting rich.
